Monika After Story/game/script-topics.rpy

```diff
@@ -14435,3 +14435,64 @@ label monika_language_nuances:
+    m 2eua "But to answer your question..."
+    m 2eud "I do believe, or at least want to believe, that there has to be {i}something{/i} out there."
+    m 2rksdla "I guess part of it has to do with me finding the idea of us being the only ones rather depressing, {w=0.3}{nw}"
+    extend 2eud "but when you think about it a little, it doesn't sound so unlikely..."
+    m 4eksdla "After all, saying the universe is vast would be {i}quite{/i} the understatement."
+    m 4eua "All you need is just one planet with the right conditions and environment to be favorable for life to develop, right?"
+    m 4esb "And there are 8 planets in our Solar System alone... {w=0.5}{nw}" 
+    extend 4eub "...But of course, there are many more Solar Systems, each with their own planets within them."
+    m 4dsc "Now, consider that our galaxy, the Milky Way, supposedly contains hundreds of billions of stars. And that's only counting one galaxy, obviously..."
+    m 4esd "Galaxies are usually held together in groups by gravity. We live in the 'Local Group', which contain about 60 galaxies."
+    m 1esd "Go further than that, and you'll start to see galaxy clusters, which are much larger groups of galaxies. The closest one from us, the 'Virgo Cluster', is estimated to contain at least a thousand galaxies."
+    m 1eud"But you can go even further, as groups and clusters of galaxies are themselves part of even bigger entities known as superclusters."
+    m 1wud "And we could keep going, as we aren't even sure of how big the universe is. Its size could be infinite, for all we know."
+    m 1lud "If it isn't, we could take into consideration the idea that there might be something beyond the boundaries of our universe."
+    
+    if renpy.seen_label('monika_clones'):
+        m 1lksdla "...Or even start talking about the multiverse theory, but I digress."
+
+    m 3ekb "I think you get the point..."
+    m 3euc "Wouldn't it be a little conceited to assume that we, human beings of planet Earth, are truly the only sentient beings in something so gigantic?"
+    m 3eud "I mean, with odds like these, surely at least {i}one{/i} other planet somewhere must be hospitable enough for life..."
+    m 1euc "...Life which might evolve to a point where it's intelligence is comparable, if not even greater, than our own."
+    m 1rsc "Although I suppose I can also understand why some people would be doubtful. It is suspicious that we're able to observe the universe so far beyond our planet, but we have yet to find any sign of life..."
+    m 1rkc "It probably doesn't help that some people are overreacting over the smallest things, like UFO footage that could very well be faked, either."
+    m 1ruc "But then again, if aliens do exist, there could also be many reasons why we haven't found them yet..."
+    m 2etc "Perhaps they're too far away for us to find them, or maybe they just don't have the technology to receive and answer our messages for now."
```

Comment:
Another possibility is that there's life out there, but the life we find might not be intelligent, or maybe there _isn't_ life nearby at the moment. Keep in mind that the earth is about 4,500,000,000 years old, while humans have only existed for about 200,000. There's no doubt that it's possible for alien life to exist, but the probability of humans existing at the same time as life we can notice is relatively low. It might be more likely to find devoid structures and the remains of a civilization than a live one.

Reply:
Good point.
I also considered talking about the fact the we would probably never be able to leave the local group due to the universe constantly expanding, but felt like this would make the topic longer than it needed to be.

I can always edit it, if you feel like it would be worth adding.